src/sprites/clone.js

```diff
@@ -106,6 +106,18 @@ Clone.prototype.visible = true;
 Clone.prototype.size = 100;
 
 /**
+ * How Far It Is Stretched On The X Axis
```

Comment:
Might want to adjust this to fit more with the other JSdoc comments - "it" is not implied ("whether THE CLONE is currently visible") and The Capitalization Could Use Some Adjusting. (Also it's missing a period at the end of the line.)

Same goes for "How Far It Is Stretched On The Y Axis".